src/kontrol/prove.py

```diff
@@ -585,6 +583,24 @@ def _init_cterm(
         'STORAGESLOTSET_CELL': set_empty(),
     }
 
+    if is_test or is_setup or is_constructor:
+        init_account_list = _create_initial_account_list(program, summary_entries)
+        init_subst_test = {
+            'OUTPUT_CELL': bytesToken(b''),
+            'CALLSTACK_CELL': list_empty(),
+            'CALLDEPTH_CELL': intToken(0),
+            'ID_CELL': Foundry.address_TEST_CONTRACT(),
+            'LOG_CELL': list_empty(),
+            'ACCESSEDACCOUNTS_CELL': set_empty(),
+            'ACCESSEDSTORAGE_CELL': map_empty(),
+            'INTERIMSTATES_CELL': list_empty(),
+            'TOUCHEDACCOUNTS_CELL': set_empty(),
+            'STATIC_CELL': FALSE,
+            'ACCOUNTS_CELL': KEVM.accounts(init_account_list),
+        }
+        for k, v in init_subst_test.items():
+            init_subst[k] = v
```

Comment:
Instead of the `for`, you can update the `dict` using  `update()`. If `init_subst` already contains keys present in `init_subst_test`, their values are also updated.
```suggestion
        init_subst.update(init_subst_test)
```

Reply:
Done.